src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -193,14 +196,19 @@ const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
         )}
         {customError && (
           <div className="mt-7">
-            <NotificationBanner icon={WarningCircle} status="error">
+            <NotificationBanner
+              icon={WarningCircle}
+              status="error"
+              testId="action-sidebar-error"
+            >
               {customError.message?.toString()}
             </NotificationBanner>
           </div>
         )}
         {flatFormErrors.length ? (
           <div className="mt-7">
             <NotificationBanner
+              testId="action-sidebar-error"
```

Comment:
I think it might be possible for this to show at the same time as the customError above, so maybe these need different testIds?